src/jit/importer.cpp

```diff
@@ -12814,10 +12814,17 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
 #ifdef FEATURE_CORECLR
                     // In coreclr the delegate transparency rule needs to be enforced even if verification is disabled
```

Comment:
You can delete this whole block instead and the impInsertCalloutForDelegate method. We are not doing any delegate transparency check in CoreCLR. It is left-over from Silverlight in a browser.